i18n/nl.json

```diff
@@ -44,6 +44,11 @@
     "notifications": "Meldingen",
     "logout": "Afmelden"
   },
+  "actions": {
+    "add": "toevoegen",
+    "previous": "Vorige",
+    "next": "Volgende"
```

Comment:
Maybe move the `previous` and `next` actions to a `pagination` object instead of `actions`. That way the context is clearer.

Reply:
the only counterargument I can give is that these "actions" can be reused for different purposes. And then we would have duplicate labels for the same thing

Reply:
@bertramakers

Reply:
I don't think having duplicate translations for different contexts is a bad thing. We don't determine the exact copy as developers, the PO does. For them, having "duplicate" translations would have the advantage of:
- Easily finding the context of a specific key (e.g. `actions.next` is ambiguous, `pagination.next` is a lot clearer)
- Being able to differentiate in the final copy if they want (e.g. `rol toevoegen`, `label toevoegen`, etc instead of one `toevoegen`)

Reply:
It's always a tough balance to find though. IMO `productions.pagination.next` would be too specific (we at least want the "next" and "previous" to be consistent in all paginations), while `actions.next` is too broad (who knows where this will be used over time).